src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig

```diff
@@ -64,7 +64,11 @@
           'data-modal-title': 'Create a new invoice'|trans({}, 'Admin.Orderscustomers.Feature'),
           'data-modal-apply': 'Create'|trans({}, 'Admin.Actions'),
           'data-modal-cancel': 'Cancel'|trans({}, 'Admin.Actions'),
-          'data-modal-body': 'Are you sure you want to create a new invoice?'|trans({}, 'Admin.Orderscustomers.Notification')
+          'data-modal-body': 'Are you sure you want to create a new invoice?'|trans({}, 'Admin.Orderscustomers.Notification'),
+          'data-modal-edit-price-title': 'Edit the price'|trans({}, 'Admin.Actions'),
+          'data-modal-edit-price-apply': 'Continue'|trans({}, 'Admin.Actions'),
```

Comment:
```suggestion
          'data-modal-edit-price-apply': 'Edit'|trans({}, 'Admin.Actions'),
```

Reply:
For consistency, thanks.

Reply:
It feels weird to click on an 'Edit' button to validate an action, from a UX perspective I would expect a form or an input to appear, this could be confusing But the consistency is important though like you said, maybe something in between 'Validate edition', 'Confirm edition' or 'Apply modification' ?

Reply:
Hi @jolelievre I see your point! I think we should keep a one-verb label for the confirmation button to stay consistent with the "_**cancel**_" button. 

I looked in the BO and there seems to be the verb "**_update_**"as a confirmation button when you edit an information. So I suggest "**_Update_**" in _**Admin.Actions**_. Wdyt?

![image](https://user-images.githubusercontent.com/64894223/89882743-84ba5400-dbc7-11ea-9d71-b8bda185c5cc.png)

Reply:
Seems perfect ;)